src/main/java/net/badbird5907/anticombatlog/AntiCombatLog.java

```diff
@@ -114,6 +114,23 @@ public static void tag(Player victim, Player attacker) {
             attacker.sendMessage(StringUtils.format(ConfigValues.getCombatTaggedMessage(), ConfigValues.getCombatTagSeconds() + ""));
     }
 
+    public static void tagSingle(Player victim) {
+        if (victim.getGameMode() == GameMode.CREATIVE)
+            return;
+        CombatTagEvent event = new CombatTagEvent(victim, victim);
```

Comment:
This could cause problems with some plugins, would recommend making a completely new event and (maybe? calling CombatTagEvent if the player gets hit in combat)